### ISSUE TITLE
To-Do 

### ISSUE THREAD

**Issue by sallamander** [sallamander/neural-networks-intro#2]
- [x] Make learning rate a parameter everywhere (instead of hard-coded) 
- [x] Move everything into chapters 
- [x] Rename repository
- [x] Try to give more insight/detail in the multiple linear regression implementations of things (or make things less verbose, since a lot of it is repeated stuff from simple linear regression)
- [x] Touch up `tensorflow` descriptions.
